logic/settings.ts

```diff
@@ -584,6 +586,21 @@ export class NovelWordCountSettingTab extends PluginSettingTab {
 						})
 				);
 
+			new Setting(containerEl)
+				.setName("Exclude markdown footnotes")
+				.setDesc(
+					"Exclude markdown footnotes as per this specification https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/"
```

Comment:
Let's update the description to:

```suggestion
					"Exclude footnotes[^1] from counts. May affect performance on large vaults."
```